### ISSUE TITLE
[Calendar] Сравнение дат

### ISSUE THREAD

**Issue by pavel-b-novikov** [yuukos/nsudotnet#1]
Я виду вот этот кусок: ```(day == DateTime.Now.Day) && (readDate.Month == DateTime.Now.Month) && (readDate.Year == DateTime.Now.Year)``` 
Я понимаю что у вас тут day различается (но все еще не понимаю почему никто не догадался впихнуть date.AddDay(1) в цикл). А вообще - можно ли сравнивать даты через > и < и через ==? И почему?

**Comment by yuukos** [comment]
Сравнивать даты с помощью операторов можно (и там еще +, -, !=, ...)

**Comment by yuukos** [comment]
А почему... А почему бы и нет?)

**Comment by pavel-b-novikov** [comment]
Ответ милый, но все же хотелось бы в деталях.

**Comment by yuukos** [comment]
@pavel-b-novikov, мой ответ подразумевал, что я не очень понимаю как ответить(

**Comment by yuukos** [comment]
У DateTime есть метод CompareTo, который сравнивает даты. Так почему бы и не определить и операторы для дат.

**Comment by yuukos** [comment]
@pavel-b-novikov и еще вопрос. А что насчет задачи LinesCounter? Просто не было issues ни об ошибках, ни о сдаче.

**Comment by pavel-b-novikov** [comment]
Нет, CompareTo это немного не то. Меня интересует, скажем так, причина по которой сравнение дат через == работает и при том работает очень быстро

**Comment by yuukos** [comment]
Полагаю, что "всему виной" то, как сравнивает оператор == - количество тактов. В точку или Акела промахнулся?
